rfcs/0006-picasso-runtime-upgrades.md

```diff
@@ -0,0 +1,55 @@
+# Overview
+
+Proposes flow to do runtime upgrades, likely similar to what others do.
+
+Expected that the reader has understood official Parity documents and explainers regarding upgrades.
+
+## Constraints
+
+- Upgrades can be done only under `Root` origin.
+- Upgrades are not immediate because runtime should be copied to the majority of para chain collators and relay validators and be enabled simultaneously on a specific block in the future.
```

Comment:
```suggestion
- Upgrades are not immediate because runtime should be copied to the majority of parachain collators and relay validators and be enabled simultaneously on a specific block in the future.
```